Sources/SyncDataProviders/Credentials/CredentialsProvider.swift

```diff
@@ -40,7 +40,6 @@ public final class CredentialsProvider: DataProvider {
     // MARK: - DataProviding
 
     public override func prepareForFirstSync() throws {
-        lastSyncTimestamp = nil
```

Comment:
We don't nullify timestamps anymore (this was part of enabling selective sync). Bookmarks Provider doesn't have this line of code, and I just forgot to remove it from CredentialsProvider, so I'm doing it now.